src/modules/m_filter.cpp

```diff
@@ -89,6 +91,9 @@ class FilterResult
 				case 'o':
 					flag_no_opers = true;
 				break;
+				case 'r':
```

Comment:
This should be at the end because otherwise old versions will break when parsing flags.